pennylane/ops/op_math/prod.py

```diff
@@ -97,6 +97,9 @@ def prod(*ops, id=None, lazy=True):
     >>> prod_op = prod(qfunc)(1.1)
     >>> prod_op
     CNOT(wires=[0, 1]) @ RX(1.1, wires=[0])
+
+
+    Notice how the order appears reversed in the output. However, this is correct because the operators are applied from right to left.
```

Comment:
```suggestion
    Notice how the order in the output appears reversed. However, this is correct because the operators are applied from right to left.
```